cjdns-hdr/src/content_type.rs

```diff
@@ -88,4 +88,4 @@ fn test_content_type_conversion() {
     for &code in &unknown_content_types {
         assert_eq!(ContentType::from_primitive(code), ContentType::Other);
     }
-}
```

Comment:
Why to change this? This file is irrelevant to the task. We've discussed that before.